evalml/pipelines/utils.py

```diff
@@ -264,48 +274,109 @@ def _make_stacked_ensemble_pipeline(
         input_pipelines (list(PipelineBase or subclass obj)): List of pipeline instances to use as the base estimators for the stacked ensemble.
             This must not be None or an empty list or else EnsembleMissingPipelinesError will be raised.
         problem_type (ProblemType): problem type of pipeline
+        final_estimator (Estimator): Metalearner to use for the ensembler.
         n_jobs (int or None): Integer describing level of parallelism used for pipelines.
             None and 1 are equivalent. If set to -1, all CPUs are used. For n_jobs below -1, (n_cpus + 1 + n_jobs) are used.
             Defaults to -1.
+        use_sklearn (bool): If True, instantiates a pipeline with the scikit-learn ensembler.
 
     Returns:
         Pipeline with appropriate stacked ensemble estimator.
     """
     parameters = {}
     if is_classification(problem_type):
-        parameters = {
-            "Sklearn Stacked Ensemble Classifier": {
-                "input_pipelines": input_pipelines,
-                "n_jobs": n_jobs,
+        if use_sklearn:
+            parameters = {
+                "Sklearn Stacked Ensemble Classifier": {
+                    "input_pipelines": input_pipelines,
+                    "n_jobs": n_jobs,
```

Comment:
Is there a reason the regression pipelines take a `final_estimator` here but the classifiers don't?

Reply:
Oops, I think it might have been left over from the previous copy/paste job. I updated this so that `final_estimator` is a param in sklearn ensemblers and not in the new one.